tests/config_test.js

```diff
@@ -48,82 +48,82 @@ module.exports = {
           directoryPrefix: 'test',
           modes: {
             driving: {
-                // set host to null when you want node to start osrm locally with the provided osrmPath and port
+                // set host to null when you want node to start osrm locally with the provided port
```

Comment:
Should we remove all this from the preference? Or only when we remove the deprecated part? @tahini  ? 

If only the deprecated part, we should add a comment to it. (And maybe an issue number like I've said ealier

Reply:
Indeed, the test config should ideally be with the new format and not keep the deprecated one.